crates/diesel_models/src/payment_attempt.rs

```diff
@@ -791,6 +791,7 @@ pub struct PaymentAttemptUpdateInternal {
     // client_version: Option<String>,
     // customer_acceptance: Option<pii::SecretSerdeValue>,
     // card_network: Option<String>,
+    pub connector_mandate_detail: Option<ConnectorMandateReferenceId>,
```

Comment:
is this required?

This should be connector_token_details or connector_mit_details